packages/contracts/test/contracts/L1/rollup/OVM_CanonicalTransactionChain.gas.spec.ts

```diff
@@ -196,4 +197,47 @@ describe('[GAS BENCHMARK] OVM_CanonicalTransactionChain', () => {
       )
     }).timeout(100000000)
   })
+
+  describe('enqueue [ @skip-on-coverage ]', () => {
+    beforeEach(() => {
+      OVM_CanonicalTransactionChain =
+        OVM_CanonicalTransactionChain.connect(sequencer)
+    })
+
+    it('cost to enqueue a transaction above the prepaid threshold', async () => {
+      const ENQUEUE_L2_GAS_PREPAID =
+        await OVM_CanonicalTransactionChain.ENQUEUE_L2_GAS_PREPAID()
+      const data = '0x' + '12'.repeat(1234)
```

Comment:
I'd recommend pulling this into `before` and using the same variable for both tests.